update: set up kapt and configure processing (#5632)
JetBrains/kotlin-web-site: docs/topics/jvm/jvm-annotation-processors.md

   ```
   
   * Add the execution of the `kapt` goal from `kotlin-maven-plugin` **before** the `compile` execution.

>    * Configure the [level of annotation processing](kapt.md#in-maven) using the `aptMode` option.

Reviewer: ```suggestion
   * Configure which [annotation processing steps kapt runs](kapt.md#in-maven) using the `aptMode` option.
```
